solidity/contracts/bridge/BridgeState.sol

```diff
@@ -251,6 +251,10 @@ library BridgeState {
         // HASH160 over the compressed ECDSA public key) to the basic wallet
         // information like state and pending redemptions value.
         mapping(bytes20 => Wallets.Wallet) registeredWallets;
+        // Reserved storage space in case we need to add more variables.
+        // See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+        // slither-disable-next-line unused-state
+        uint256[17] __gap;
```

Comment:
Yep yep I have read the OZ docs. My comment was about making it explicit so there will be no need to dig into OZ docs every time.

Reply:
@lukasz-zimnoch isn't the current comment and a reference to OZ docs enough? It's all described in the doc, I'm not sure what to add more here.

Reply:
> @nkuba I am counting storage slots occupied in the `main` version of this file and I think 29 slots are occupied so we should have a 21 slots gap. But... I think this structure deserves an exception of `uint256[50] __gap`. All `Bridge` state is stored here, no library has its own storage. It's different from what we do in Random Beacon and ECDSA. Taking into account our plans for v2.1 (instaminting for example), I think 50-slots gap is required. It's not about bugfixing or small new features, it's about adding major new features.

ba8fb5ce26c9e246d59ee7ef8579153a5f5aac51

Reply:
I meant something like:
```
// Reserved storage space in case we need to add more variables. The convention from OpenZeppelin says that
// the storage space should add up to 50 storage slots. There is 29 slots occupied by the `Storage` struct fields 
// hence the gap has 21 slots.
// See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
// slither-disable-next-line unused-state
```
It was not obvious to me at the first sight, even after reading the OZ docs. This is why having it noted explicitly may be helpful for others.

Reply:
39b02b7b0828d64bccfe5b20c0e41ad1fced6be5